### PR TITLE
Refactor common Flags

### PR DESCRIPTION
**Pull request by acidmoose**
- "service-name" and "service-environment" are used in both
the logging and consul modules.
- Refactored to be available from SecondBase.

### CODE REVIEW

File: core/src/main/java/org/secondbase/core/config/SecondBaseConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.secondbase.core.config;
+
+/**
```
Comment (neumayer) [inline core/src/main/java/org/secondbase/core/config/SecondBaseConfiguration.java]: This comment. Does not need to be here.

**Review by neumayer — APPROVED** [review]
I will try this in my experiments.

File: core/src/main/java/org/secondbase/core/SecondBase.java [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
 import org.secondbase.core.config.SecondBaseModule;
+import org.secondbase.flags.Flag;
 import org.secondbase.flags.Flags;
 
 /**
  * Coordination class for SecondBase modules. Handles Flags parsing and cooperation between modules.
  */
 public class SecondBase {
 
+    @Flag(
```
Comment (neumayer) [inline core/src/main/java/org/secondbase/core/SecondBase.java:12]: Those are not flags-registered? And as such fail when running the example. Did you forget to check in changes in the SecondBase class?

Comment (acidmoose) [reply core/src/main/java/org/secondbase/core/SecondBase.java:12]: just forgot to add the flags

File: core/src/main/java/org/secondbase/core/config/SecondBaseConfiguration.java (hunk 1 above)
Comment (acidmoose) [reply core/src/main/java/org/secondbase/core/config/SecondBaseConfiguration.java]: done

**Pull request merged**